go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -269,15 +293,27 @@ func (ovn *Controller) deleteLoadBalancerRejectACL(lb, vip string) {
 			return
 		}
 		aclUUID, err = ovn.findStaleRejectACL(lb, ip, port)
-		if err == nil {
-			ovn.removeACLFromPortGroup(lb, aclUUID)
+		if err != nil {
+			klog.Infof("Unable to delete Reject ACL for load-balancer: %s, vip: %s. No entry in cache and "+
```

Comment:
log message needs update?

Reply:
looks OK to me

Reply:
It's trying to findStaleReject, not deleting it, so probably  "Unable to find stale reject ACL....." would be better ?